README.md

```diff
@@ -31,22 +31,16 @@ An example configuration file can be found in `erldns.example.config`. Copy it t
 
 ## Running
 
-### Launch directly:
-
-```bash
-overmind start
```

Comment:
I don't *think* we want to mention overmind, which seems to be a tool that requires auth and whatnot. Am I missing something?

Reply:
No wonder I was confused... I thought this was about [*this* overmind](https://github.com/overmindtech/cli) CLI tool. 🤦 

Ok, makes sense then. Yep, I’m familiar with `Procfile`s and all. I'll make sure that we have `make start` here and maybe a note about overmind then. I'll close this PR out.